packages/server/lib/config.ts

```diff
@@ -138,7 +138,14 @@ export function mergeDefaults (
     config.baseUrl = url.replace(/\/\/+$/, '/')
   }
 
-  const defaultsForRuntime = configUtils.getDefaultValues(options)
+  let defaultsForRuntime = configUtils.getDefaultValues(options)
+
+  // As we normalize the config based on the selected testing type, we need
+  // to do the same with the default values to resolve those correctly
+  defaultsForRuntime = {
+    ...defaultsForRuntime,
+    ...defaultsForRuntime[options.testingType],
+  }
```

Comment:
imo, this logic would be more suitably moved to `packages/config/src/index.ts` where `getDefaultValues` is defined. this is the only usage of that function, so IMO it would make sense to confine this logic to the `config` package. what do you think @estrada9166 ?

Reply:
Sounds good! yeah, I'm gonna update it!